po/de.po

```diff
@@ -13250,28 +13250,27 @@ msgstr "Option \"%s\" wird ignoriert für %s\n"
 
 #: builtin/fetch.c:1414
 msgid "multiple branch detected, incompatible with --set-upstream"
-msgstr ""
+msgstr "mehrfache Branch erkannt, inkompatibel mit --set-upstream"
 
 #: builtin/fetch.c:1429
-#, fuzzy
 msgid "not setting upstream for a remote remote-tracking branch"
-msgstr "Upstream-Branch '%s' nicht als Remote-Tracking-Branch gespeichert"
+msgstr "Upstream wird nicht als entfernte Remote-Tracking-Branch gesetzt"
```

Comment:
Ich würde vielleicht schreiben: "Setze keinen Upstream für einen entfernten Remote-Tracking-Branch."

Reply:
"Setze Upstream nicht für einen entfernten Remote-Branch." ?
not setting = setze nicht
"not" ist ja nicht fü Upstream gemeint?
Ich schau aber auch nochmal wegen der Verwendung dieses Textes innerhalb des Quellcodes nach...

Reply:
Ich denke mein Vorschlag passt so.

Die Fehlermeldung ist für folgende Situation:
Ich fetche von einem Remote-Repository und in diesem Repository gibt es Branches in "refs/heads/*", quasi die lokalen Branches, und Remote-Tracking-Branches z.B. für "origin" in "refs/remotes/origin/*". So wie man es in einem lokalen Repo üblicherweise hat.
Wenn man --set-upstream nutzt, darf man nur die lokalen Branches des Remote-Repos verwenden, also alles unter "refs/heads/*", nicht jedoch die Remote-Tracking-Branches.
Also:
git fetch --set-upstream origin master # OK
git fetch --set-upstream origin refs/heads/master # OK
git fetch --set-upstream origin refs/remotes/origin/master # NOK